src/components/block-gallery/shared/transforms.js

```diff
@@ -25,9 +25,9 @@ function GalleryTransforms( props ) {
 		gridSize: props.gridSize,
 		gutterMobile: props.gutterMobile,
 		height: props.height,
-		images: props.images.map( ( image, index ) => {
-			return { ...helper.pickRelevantMediaFiles( image ), index };
-		} ),
+		images: props.images?.map((image, index) => {
+			return { ...helper.pickRelevantMediaFiles(image), index };
+		}),
```

Comment:
@glendaviesnz Thanks for the PR! Would you mind just re-adding the spaces that were removed on these three lines?

Reply:
Sorry, not sure how I overlooked that - fixed now.